website/templates/project/addon/widget.mako

```diff
@@ -9,14 +9,17 @@
 
                 </div>
             </div>
+    % else:
+        <div>
+    % endif
 
     % if complete:
 
         <div class="panel-body">
             ${self.body()}
         </div>
 
-    % else:
+    % elif not complete and 'admin' in user['permissions']:
```

Comment:
the block should be showing if the addon is properly configured or the current user has permission to configure the addon. So I would think that as long as the user has write permission or it is complete, it should be displayed.